client/my-sites/themes/themes-selection.jsx

```diff
@@ -143,3 +157,31 @@ export default connect(
 		};
 	}
 )( ThemesSelection );
+
+class ThemesSelectionWithPage extends React.Component {
```

Comment:
Containing this stuff closer to `ThemesSelection` (instead of adding state and methods for `page` management to `ThemeShowcase`) is really nice. I wonder if we should add a small TODO that we'll attempt to keep it all inside one component at some point? (Are we? 😄 )